docs/source/advanced/mixed_precision.rst

```diff
@@ -5,49 +5,154 @@
 
 .. _amp:
 
-Mixed Precision Training
-========================
-
-Mixed precision combines the use of both FP32 and lower bit floating points (such as FP16) to reduce memory footprint during model training, resulting in improved performance.
 
-Lightning offers mixed precision training for GPUs and CPUs, as well as bfloat16 mixed precision training for TPUs.
+#########
+Precision
+#########
+
+There are numerous benefits to using numerical formats with lower precision than the 32-bit floating-point or higher precision such as 64-bit floating-point.
+
+Lower precision, such as the 16-bit floating-point, enables the training and deployment of large neural networks since they require less memory, enhances data transfer operations since they require
```

Comment:
To me this sounds like large models would in general require less memory than smaller models and not that this is due to fp16

Reply:
true. updated!